src/DynamicRouter.php

```diff
@@ -259,8 +230,10 @@ public function match($pathinfo)
     public function matchRequest(Request $request)
     {
         if ($this->eventDispatcher) {
+            $eventName = Events::PRE_DYNAMIC_MATCH_REQUEST;
```

Comment:
lets inline the constant into the `doDispatch` call

Reply:
Good point, fixed for both parameters and both `doDispatch` calls!